mandatory/1-fix-functions.js

```diff
@@ -21,9 +19,9 @@ function getMood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = 10;
```

Comment:
The value of isBigEnough is declared = num and greater then 10.
let isBigEnough = num > 10